playlist.py

```diff
@@ -0,0 +1,91 @@
+from PyQt5.QtWidgets import QDialog, QFileDialog, QAbstractItemView
+from playlist_ui import Ui_Dialog
+from clip import load_song_from_file, verify_ext
+import json
+from os.path import basename, splitext
+
+
+def getSongs(file_names):
+    r = []
+    for f in file_names:
+        try:
+            r.append(load_song_from_file(f))
```

Comment:
actually @sonejostudios opted for your original solution (see sonejostudios/superboucle#5) - if you have complex songs with big hq samples memory usage might become an issue, and the plan is to be able to use the playlist on stage ... so I'm not sure about how to deal with that ... but whatsoever, I would say its not a top-priority issue.